tests/lcs/core/interface/image-undo.livecodescript

```diff
@@ -0,0 +1,58 @@
+script "CoreEngineClipboard"
```

Comment:
@runrevmark yes but I'm running the tests on linux to make sure they'll also run on travis. that's why I have them on git. I do not see them as the final version